popSimManu.tex

```diff
@@ -163,10 +172,15 @@ \section*{Introduction}
 %ADK: made some edits here
 %RNG: Removed the sap. :-P Feel free to put it back if you like it better.
 %JK: not crazy about it either. Making it shorter helps though?
+%ALG: I think it sounds fine.
+%ALG: Do we also need to say something about how it’s useful to people not in methods development? 
 
 It has been challenging to assess the degree to which modeling assumptions
 and choices of data summaries can affect the ultimate results of population
-genetic inference. Yet there
+genetic inference. 
+%ALG: Do we want to say here that it is expected to differ based on modeling assumptions 
+% and stats and why it is expected, before giving an example?
```

Comment:
Hm - yes, kinda - but that seems like a bit much to explain here.

Reply:
that's fine. maybe keep in mind for later.